vite/src/content/LandingPage.jsx

```diff
@@ -5,7 +5,7 @@ export const LandingPage = () => {
     return (
         <div className="m-8 flex flex-col align-center">
             <h2 className='mt-8 text-3xl font-bold place-self-center'>What is <span className="bg-gradient-to-r from-emerald-600 to-green-300 bg-clip-text text-transparent">ECOWatch?</span></h2>
-            <p className='my-4 place-self-center flex-wrap'>
+            <p className='my-4 place-self-center flex-wrap w-[75%]'>
```

Comment:
i believe the scemantic way to do this is `w-9` (because tailwind divides the screen into 12ths)
you don't need to fix this though tbh